src/components/tile/tile.style.ts

```diff
@@ -76,10 +77,13 @@ const TileContent = styled.div<TileContentProps>`
       }
     `}
 
-    ${width &&
+    ${(width || height) &&
     css`
       flex-grow: 0;
-      ${computeWidth({ width })}
+      ${computeSizing({
+        width: width || /* istanbul ignore next */ undefined,
```

Comment:
This is fully tested in cypress so ignored here, implementation is likely to slightly change as part of FE-5681 but it was preferred that this prop was surfaced before the breaking change is made